src/shared/reducers/map.js

```diff
@@ -0,0 +1,29 @@
+import * as types from '../actions/ActionTypes';
```

Comment:
Do not use wild card imports: https://github.com/airbnb/javascript#modules--no-wildcard